sap/adt/aunit.py

```diff
@@ -297,6 +303,14 @@ def is_warning(self):
 
         return self.severity == AlertSeverity.TOLERABLE
 
+    @property
+    def is_skipp(self):
+        """Returns true if the alert a message about skipped test.
+           ABAP: cl_abap_unit_assert=>skip( 'Skipped because of a wrong system' ).
+        """
+
+        return self.severity == AlertSeverity.TOLERABLE and self.kind == AlertKind.ABORTION
```

Comment:
Why don't we mark skipped findings that are `not TOLERABLE`? Shouldn't we skip all findings that have kind equal to `abortion`? Does this logic come from `cl_abap_unit_assert=>skip` abap code mentioned in doc string?

Reply:
As mentioned in the commit message:
`I have decided to consider skipped only the tests with the kind ABORTION and the severity TOLERABLE - I consulted ABAP code of cl_abap_unit_assert=>skip( ).`

Reply:
Also added:
```
To be defensive: I would rather receive a complain that we do not 
recognize skipped test than that we missed a real error and incorrectly
marked it as skipped.
```

Reply:
Makes sense. Thank's.